codes.md

```diff
@@ -550,7 +535,7 @@ further embellishment.
     - [Session 009](https://osf.io/es7w6/) - originally text files labeled
       with "txt" but participant says they stop labeling because of laziness
 
-### D2. People run into constraints with their existing sketch.
+### D2. Participants ran into constraints with their existing sketch.
```

Comment:
I may want to add Session 030 and 033 -- both had difficulty picking icons/file extensions for the code files that also "went with" the existing icons/encoding. Possibly also people in the junk drawer who drew pens and pencils and then were concerned about differentiating sharpies. 

This is a Katy TODO tomorrow (add 2-3 instances of each of those)